package.json

```diff
@@ -20,7 +20,9 @@
     "url": "https://github.com/apollostack/graphql-tag/issues"
   },
   "homepage": "https://github.com/apollostack/graphql-tag#readme",
-  "dependencies": {},
+  "dependencies": {
+    "@types/graphql": ">=0.8.6 <=0.11.7"
```

Comment:
0.12.x should also export `DocumentNode`, how about `>=0.8.6`

Reply:
At the time I opened this PR 0.12 didn't exist, will update.
I don't understand the second question? 0.8.6 was the first version to export it

Reply:
I meant that by just specifying the dependency as `"@types/graphql": ">=0.8.6"`, and omitting the `<=x.x.x` the right hand side won't continue to need to be bumped each time a release is cut.

Reply:
How do you know the next version does not contain a breaking change?